content/patterns/alert/examples/alert.html

```diff
@@ -20,85 +20,84 @@
   <body>
     <nav aria-label="Related Links" class="feedback">
       <ul>
-        <li><a href="https://github.com/w3c/aria-practices/projects/20">Related Issues</a></li>
-        <li><a href="../alert-pattern.html">Design Pattern</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/projects/20">関連するIssues</a></li>
+        <li><a href="../alert-pattern.html">デザインパターン</a></li>
       </ul>
     </nav>
     <main>
-      <h1>Alert Example</h1>
+      <h1>アラートの例</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section>
-        <h2>About This Example</h2>
+        <h2>この例について</h2>
         <p>
-          The below example demonstrates the <a href="../alert-pattern.html">Alert Pattern</a>.
-          Activating the <q>Trigger Alert</q> button causes a message to be inserted into the example alert element.
+          以下の例は、<a href="../alert-pattern.html">アラート パターン</a>のデモンストレーションです。<q>アラートをトリガーする</q>ボタンを作動すると、例のところにあるアラートの要素にメッセージが挿入されます。
         </p>
-        <p>Similar examples include:</p>
+        <p>類似の例には以下があります：</p>
         <ul>
-          <li><a href="../../alertdialog/examples/alertdialog.html">Alert Dialog Example</a>: A confirmation prompt that demonstrates an alert dialog.</li>
+          <li><a href="../../alertdialog/examples/alertdialog.html">アラートダイアログの例</a>: アラートダイアログのデモンストレーションである確認プロンプトです。</li>
```

Comment:
動詞の demonstrate をどう訳すか、なやましいですね。「実演する」みたいな訳もありますが。ちょっと意訳して「アラートダイアログによる確認プロンプトのデモ」くらいにしたほうがわかりやすさはありそうです。

```suggestion
          <li><a href="../../alertdialog/examples/alertdialog.html">アラートダイアログの例</a>: アラートダイアログによる確認プロンプトのデモです。</li>
```